lib/connection.ts

```
@@ -1,13 +1,46 @@
-import { Client } from 'appwrite';
+import { Client } from 'node-appwrite';
 
 class AppwriteConnection {
     private static client: Client;
 
-    static initialize(endpoint: string, project: string, apiKey: string) {
+    static initialize_(endpoint: string, project: string, apiKey: string) {
+
+        if (this.client) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)